src/Payments.sol

```diff
@@ -143,6 +144,12 @@ contract Payments is
         _;
     }
 
+    /// @notice Updates the approval status and allowances for an operator.
+    /// @param token The ERC20 token address for which the approval is being set.
+    /// @param operator The address of the operator whose approval is being modified.
+    /// @param approved Whether the operator is approved (true) or not (false) to create new rails>
+    /// @param rateAllowance The maximum payment rate the operator can set across all rails created by the operator. If this is less than the current payment rate, the operator will only be able to reduce rates until they fall below the target.
```

Comment:
"The maximum payment rate the operator can set across all rails created by the operator. where message sender is the payer of the rail*"

Reply:
Basically nice to document that these approvals are created by the message sender here for operators who create rails where the message sender is the client/payer.